### PR TITLE
fix: limits to camera calculated based on model

### PR DESCRIPTION
**Pull request by samjosephgates**
### Testing 🧪
 > [ Add any special instructions needed to test or view your changes such as environment URLs or other config details. ]

### Checklist ✔️
- [ ] Linked associated issue (if present)
- [ ] Added relevant labels
- [ ] Requested developer reviews
- [ ] Request UI review from design / PM
- [ ] Verify all code checks are passing

### CODE REVIEW

File: src/Components/3DV/SceneView.tsx [hunk 1]
```diff
@@ -372,10 +372,14 @@ function SceneView(props: ISceneViewProps, ref) {
                         );
 
                         camera.attachControl(canvas, false);
-                        camera.lowerRadiusLimit = 0;
                         cameraRef.current = camera;
                         cameraRef.current.zoomOn(meshes, true);
                         cameraRef.current.radius = radius;
+                        cameraRef.current.lowerRadiusLimit = 0;
```
Comment (msnyder-msft) [inline src/Components/3DV/SceneView.tsx:378]: can we add a comment to the properties that help with the clipping so we remember?

Comment (msnyder-msft) [reply src/Components/3DV/SceneView.tsx:378]: also a note on why *3 and *5?

**Review by msnyder-msft — APPROVED** [review]

File: src/Components/3DV/SceneView.tsx [hunk 2]
```diff
@@ -372,10 +372,14 @@ function SceneView(props: ISceneViewProps, ref) {
                         );
 
                         camera.attachControl(canvas, false);
-                        camera.lowerRadiusLimit = 0;
                         cameraRef.current = camera;
                         cameraRef.current.zoomOn(meshes, true);
```
Comment (coryboyle) [inline src/Components/3DV/SceneView.tsx:376]: Not sure if it helps, but if you set the zoomOn bool to false the camera will auto adjust the maxZ to include the bounds of the object.

File: src/Components/3DV/SceneView.tsx (hunk 1 above)
Comment (samjosephgates) [reply src/Components/3DV/SceneView.tsx:378]: Added comments and made the values make a little more sense

File: src/Components/3DV/SceneView.tsx (hunk 2 above)
Comment (samjosephgates) [reply src/Components/3DV/SceneView.tsx:376]: You can set the bool on zoomOn, but that then clips when the user zooms out, so we need to set the MaxZ to be higher than the upperRadiusLimit. Updated code to make it clearer what its doing.

**Review by darsney — APPROVED** [review]

**Pull request merged**